staging/src/k8s.io/api/core/v1/resource.go

```diff
@@ -35,6 +35,11 @@ func (rl *ResourceList) Memory() *resource.Quantity {
 	return rl.Name(ResourceMemory, resource.BinarySI)
 }
 
+// Swap returns the Swap limit if specified.
+func (rl *ResourceList) Swap() *resource.Quantity {
+	return rl.Name(ResourceSwap, resource.BinarySI)
+}
```

Comment:
I wouldn't expect this if `swap` isn't being exposed in pod specs

Reply:
Removed.

Reply:
still needs to be removed?

Reply:
It should not be removed as it is used for node.

Reply:
This is moved to node package.